core/domain/app_feedback_report_services_test.py

```diff
@@ -0,0 +1,147 @@
+        'automatically_update_topics': False,
+        'is_admin': False
+    }
+    WEB_REPORT_INFO = {
+        'user_feedback_other_text_input': 'add an admin'
+    }
+    ANDROID_REPORT_INFO_SCHEMA_VERSION = 1
+    WEB_REPORT_INFO_SCHEMA_VERSION = 1
+
+    def setUp(self):
+        super(AppFeedbackReportServicesUnitTests, self).setUp()
+        self.signup(self.USER_EMAIL, self.USER_USERNAME)
+        self.user_id = self.get_user_id_from_email(self.USER_EMAIL)
+
+        model_class = app_feedback_report_models.AppFeedbackReportModel
+        self.web_report_id = model_class.create(
+            self.PLATFORM_WEB, self.REPORT_SUBMITTED_TIMESTAMP,
+            self.REPORT_TYPE_SUGGESTION, self.CATEGORY_OTHER,
+            self.PLATFORM_VERSION, self.DEVICE_COUNTRY_LOCALE_CODE_INDIA,
+            self.ANDROID_SDK_VERSION, self.ANDROID_DEVICE_MODEL,
+            self.ENTRY_POINT_NAVIGATION_DRAWER, None, None, None, None,
+            self.TEXT_LANGUAGE_CODE_ENGLISH, self.AUDIO_LANGUAGE_CODE_ENGLISH,
+            None, self.WEB_REPORT_INFO)
+        self.android_report_id = model_class.create(
+            self.PLATFORM_ANDROID, self.REPORT_SUBMITTED_TIMESTAMP,
+            self.REPORT_TYPE_SUGGESTION, self.CATEGORY_OTHER,
+            self.PLATFORM_VERSION, self.DEVICE_COUNTRY_LOCALE_CODE_INDIA,
+            self.ANDROID_SDK_VERSION, self.ANDROID_DEVICE_MODEL,
+            self.ENTRY_POINT_NAVIGATION_DRAWER, None, None, None, None,
+            self.TEXT_LANGUAGE_CODE_ENGLISH, self.AUDIO_LANGUAGE_CODE_ENGLISH,
+            self.ANDROID_REPORT_INFO, None)
+
+    def test_scrub_android_report(self):
+        expected_report_dict = {
+            'package_version_code': 1,
+            'language_locale_code': 'en',
+            'entry_point_info': {
+                'entry_point_name': 'crash',
+            },
+            'text_size': 'MEDIUM_TEXT_SIZE',
+            'download_and_update_only_on_wifi': True,
+            'automatically_update_topics': False,
+            'is_admin': False
+        }
+
+        app_feedback_report_services.scrub_report(
+            self.android_report_id, 'scrubber_user')
+        scrubbed_report_model = (
+            app_feedback_report_models.AppFeedbackReportModel.get(
+                self.android_report_id))
+        self.assertEqual(scrubbed_report_model.scrubbed_by, 'scrubber_user')
+        self.assertEqual(
+            scrubbed_report_model.android_report_info, expected_report_dict)
+
+    def test_scrub_web_report(self):
+        app_feedback_report_services.scrub_report(
+            self.web_report_id, 'scrubber_user')
+        scrubbed_report_model = (
+            app_feedback_report_models.AppFeedbackReportModel.get(
+                self.web_report_id))
+        expected_report_dict = {}
+
+        self.assertEqual(scrubbed_report_model.scrubbed_by, 'scrubber_user')
+        self.assertEqual(
+            scrubbed_report_model.web_report_info, expected_report_dict)
```

Comment:
I'm not sure it's obvious why the expected report dict is empty--suggest adding a comment providing some context.

Reply:
Done, also added the case where the expected dict shouldn't be empty